types/matter.d.ts

```diff
@@ -642,7 +642,7 @@ declare namespace Matter {
          * @param {body} body
          * @param {number} inertia
          */
-        static setInertia(body: Body, interna: number): void;
+        static setInertia(body: Body, inertia: number): void;
```

Comment:
This is the closest thing to an api change in this PR.

Reply:
The Matter defs files come from the official matter repo (we didn't create them), so I'm happy to include this change, but it needs making upstream really.